openedx/features/subscriptions/models.py

```diff
@@ -0,0 +1,49 @@
+import logging
+
+from datetime import date
+from django.contrib.auth.models import User
+from django.db import models, transaction
+from model_utils.models import TimeStampedModel
+
+from student.models import CourseEnrollment
+
+class UserSubscription(TimeStampedModel):
+    """
+    The "UserSubscription" model.
+    """
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    subscription_id = models.PositiveIntegerField()
```

Comment:
This should be `subscription_uuid` instead of id.
and add `db_index=True`

Reply:
@ziafazal A subscription does not have a uuid. It does however has a sku. If u want me to use that instead of this. @zubair-arbi mentioned this as well but as there was no uuid as it is a simple `Product`, he said this is fine for now.